### ISSUE TITLE
Pytest-redis 3.1.0 factories are broken

### ISSUE THREAD

**Issue by petr-stefan-szn** [ClearcodeHQ/pytest-redis#679]
### What action do you want to perform

```py
from pytest_redis import factories

redis_proc = factories.redis_proc(host="127.0.0.1", port=26379)
```

### What are the results

```py
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'pytest_redis.factories' has no attribute 'redis_proc'
```

### What are the expected results

I'd like to import the `redis_proc` from `factories` successfully just like in the version 3.0.2

cc @fizyk

**Comment by dkliban** [comment]
We are getting errors in our unit tests
```
E           mirakuru.exceptions.ProcessExitedWithError: The process invoked by the <pytest_redis.executor.process.RedisExecutor: "/usr/bin/redis-server --daemonize no --rdbcompression no --rdbchecksum no --appendonly no --databases 8 --timeout 30 --pidfile /tmp/pytest-of-pulp/pytest-0/pytest-redis-redis_proc0/redis-server.31926.pid --unixsocket /tmp/pytest-of-pulp/pytest-0/pytest-redis-redis_proc0/redis.31926.sock --dbfilename dump.31926.rdb --logfile /tmp/pytest-of-pulp/pytest-0/pytest-redis-redis_proc0/redis-server.31926.log --loglevel notice --syslog-enabled no --bind 127.0.0.1 --port 31926 --dir /tmp/pytest-of-pulp/pytest-0/pytest-redis-redis_proc0 --loadmodule ''" 0x7f3fdfd0fdf0> executor has exited with a non-zero code: 1.```

**Comment by fizyk** [comment]
Sorry for that.

**Comment by petr-stefan-szn** [comment]
Thank you for the fix! A new release would be nice :slightly_smiling_face:

**Comment by fizyk** [comment]
@dkliban which pytest version are you using? I believe your bug will be addressed either by #678 if you're using pytest 7.x
